dotnetflix.Web/Components/TicketDetail.razor

```diff
@@ -64,19 +77,53 @@ else
                 var ticketTypes = await TicketService.GetTicketTypes();
                 if (ticketTypes != null) TicketType = ticketTypes.FirstOrDefault(t => t.Id == Ticket.TicketTypeId)?.Name;
             }
+
+            // Generate QR code for ticket URL
+            GenerateQRCode(TicketUrl);
         }
     }
 
+    private string TicketUrl => $"{MyNavigationManager.Uri}";
+
+    private void GenerateQRCode(string text)
+    {
+        // Create QR code data
+        QRCodeGenerator qrGenerator = new QRCodeGenerator();
+        QRCodeData qrCodeData = qrGenerator.CreateQrCode(text, QRCodeGenerator.ECCLevel.Q);
+
+        // Calculate QR code image dimensions
```

Comment:
Image is wel erg lage resolutie. Kan die niet 400px zijn bijvoorbeeld en dan tonen op 200px voor retina.

Reply:
Op dit moment worden de afmetingen gebaseerd op het aantal blokjes nodig om de tekst als QR te intepreteren. Hoe meer tekst, hoe groter de width/hight. Image stretchen maakte het niet fraaier. Ik ga nog uitzoeken of er betere dimensies aan de code gegeven kan worden.